.github/workflows/docker.yml

```diff
@@ -67,21 +67,28 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
       - name: Login to GHCR
-        if: github.event_name != 'pull_request'
+        #if: github.event_name != 'pull_request'
```

Comment:
Does this mean it will run on every commit?

Reply:
I was just doing this to test the logic. I need to remove before merging :)

Reply:
(it has been removed)